### ISSUE TITLE
errors found in adaptivesoftmax

### ISSUE THREAD

**Issue by nanfeng1101** [johntrimble/adaptive-softmax-keras#5]
First, nice work. I reimplement you code with tensorflow, but you have errors in line 154, 167。
In line 154, it's wrong to write a = tf.add(a, b) in tensorflow, it is tf.add(a, b);
In line 167, child_idx = cutsoff[0] + i 。

**Comment by johntrimble** [comment]
Sorry, I didn't get an email notification about this for some reason and missed it. Thank you for catching these errors, will patch.

**Comment by johntrimble** [comment]
> it's wrong to write a = tf.add(a, b) in tensorflow, it is tf.add(a, b);

The [docs](https://www.tensorflow.org/versions/r1.9/api_docs/python/tf/add) indicate the following usage is appropriate: `a = tf.add(a, b)`.

I see in newer versions that it has been moved to a math package though.
